classes/Media.php

```diff
@@ -624,12 +624,9 @@ public static function ieCssSplitter($compiled_css, $cache_path, $refresh = fals
         $protocol_link = Tools::getCurrentUrlProtocolPrefix();
         //return cached css
         if (!$refresh) {
-            $cached_files = scandir($cache_path);
-            foreach ($cached_files as $file) {
-                if ($file != '.' && $file != '..') {
-                    $css_url = str_replace(_PS_ROOT_DIR_, '', $protocol_link.Tools::getMediaServer('').$cache_path.DIRECTORY_SEPARATOR.$file);
-                    $splitted_css[$css_url] = 'all';
-                }
+            foreach (array_diff(scandir($cache_path), array('..', '.')) as $file) {
```

Comment:
Please see [this poc](https://www.google.it/url?sa=t&source=web&rct=j&url=http://stackoverflow.com/questions/1891917/does-using-a-function-in-foreach-loop-caches-the-result-or-calls-the-function-e&ved=0ahUKEwjk84G6rsbMAhXFfhoKHfj7COAQFggbMAA&usg=AFQjCNH4H3E9QzBGe8Y5mm0_zjTA2Roi3Q&sig2=63k5_U8xFu-tF3kuOoqAlg)